packages/bpk-component-heading/src/BpkHeading.js

```diff
@@ -24,19 +26,28 @@ import STYLES from './BpkHeading.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkHeading = props => {
-  const classNames = [getClassName(`bpk-heading-${props.level}`)];
+type Props = {
+  children: any,
+  level: string,
+  className: ?string,
+  id: ?string,
+  bottomMargin: boolean,
+};
+
+const BpkHeading = (props: Props) => {
+  const { children, level, className, id, bottomMargin } = props;
+  const classNames = [getClassName(`bpk-heading-${level}`)];
 
-  if (props.className) {
-    classNames.push(props.className);
+  if (className) {
+    classNames.push(className);
   }
-  if (!props.bottomMargin) {
+  if (!bottomMargin) {
     classNames.push(getClassName('bpk-heading--no-bottom-margin'));
   }
 
   return (
-    <props.level className={classNames.join(' ')} id={props.id}>
-      {props.children}
+    <props.level className={classNames.join(' ')} id={id}>
```

Comment:
Kept this as `props.level` as changing it to `level` an error in the console is thrown that `<level>` is not a valid html component and therefore doesn't create the component correctly

Reply:
I think this is fine, but for future reference there's a trick/hack you can use. If you make a copy of the variable that's capitalised, it shouldn't complain:

```
const Level = level
```

Reply:
Did the above suggestion and seems to be ok